src/views/edit/edit-package-view.vue

```diff
@@ -1131,6 +1131,28 @@
           loading.stopLoading()
         }
       },
+      reset () {
+        this.packageItem.id = undefined
+        this.packageItem.name = undefined
+        this.packageItem.source = undefined
+        this.packageItem.status = undefined
+        this.packageItem.descriptionURL = undefined
+        this.packageItem.description = undefined
+        this.packageItem.scope = undefined
+        this.packageItem.global = undefined
+        this.packageItem.globalNote = undefined
+        this.packageItem.contentType = undefined
+        this.packageItem.consistent = undefined
+        this.packageItem.breakable = undefined
+        this.packageItem.fixed = undefined
+        this.packageItem.listStatus = undefined
+        this.packageItem.editStatus = undefined
+        this.packageItem.ids = []
+        this.packageItem.provider = undefined // organisation
+        this.packageItem.nominalPlatform = undefined
+        this.allNames = { name: undefined, alts: [] }
+        this.reload()
```

Comment:
When doing this on an existing package, it causes the stepper (step 2) to flash red (invalid) if reset while on this exact step, due to to the value of the computed property `currentStepValid` temporarily becoming `false` before the reload, since all of the required fields are set to `undefined`. To prevent this, we could start the `loading.startLoading()` at the start of `reset()` and then check  `loading.isLoading()` for `currentStepValid` as well.